sql/2022/javascript/lighthouse_unminified_js.sql

```diff
@@ -0,0 +1,18 @@
+#standardSQL
+# Pages with unminified JS
+SELECT
+  score,
+  COUNT(0) AS pages,
+  SUM(COUNT(0)) OVER () AS total,
+  COUNT(0) / SUM(COUNT(0)) OVER () AS pct
+FROM (
+  SELECT
+    JSON_EXTRACT_SCALAR(report, "$.audits['unminified-javascript'].score") AS score
+  FROM
+    `httparchive.lighthouse.2022_06_01_mobile`)
```

Comment:
All LH queries should utilize the new desktop data (also group by `_TABLE_SUFFIX AS client`)
```suggestion
    `httparchive.lighthouse.2022_06_01_*`)
```